integration/tests/verification/verification_test.go

```diff
@@ -0,0 +1,110 @@
+package verification
```

Comment:
No new code, only consolidates all integration tests of verification node into a single test suite.